semantic-conventions/README.md

```diff
@@ -59,6 +80,106 @@ const span = tracer.startSpan(spanName, spanOptions)
+console.log(DBSYSTEMVALUES_POSTGRESQL); // migrate to 'DB_SYSTEM_VALUE_POSTGRESQL' (in incubating [*])
+```
+
+See [Migrated usage](#migrated-usage) below.
+
+### Migrating from `SemanticAttributes.*`, `SemanticResourceAttributes.*`, ...
+
+Deprecated as of `@opentelemetry/semantic-conventions@1.0.0`.
+
+Before v1.0.0, constants were exported in namespace objects `SemanticResourceAttributes` and `SemanticAttributes`, and a namespace object for enumerated values for some fields (e.g. `DbSystemValues` for values of the 'db.system' enum). For example:
+
+**Deprecated usage:**
+
+```js
+import {
+  SemanticAttributes,
+  SemanticResourceAttributes,
+  DbSystemValues,
+} from '@opentelemetry/semantic-conventions';
+
+// 'service.name' resource attribute
+console.log(SemanticResourceAttributes.SERVICE_NAME); // migrate to 'ATTR_SERVICE_NAME'
+
+// 'http.route' attribute
+console.log(SemanticAttributes.HTTP_ROUTE); // migrate to 'ATTR_HTTP_ROUTE'
+
+// 'db.system' attribute
+console.log(SemanticAttributes.DB_SYSTEM); // migrate to 'ATTR_DB_SYSTEM' (in incubating [*])
+
+// 'postgresql' enum value for 'db.system' attribute
+console.log(DbSystemValues.POSTGRESQL); // migrate to 'DB_SYSTEM_VALUE_POSTGRESQL' (in incubating [*])
+```
+
+See [Migrated usage](#migrated-usage) below.
+
+### Migrated usage
+
+```js
+import {
+  ATTR_SERVICE_NAME,
+  ATTR_HTTP_ROUTE,
+  METRIC_HTTP_CLIENT_REQUEST_DURATION
+} from '@opentelemetry/semantic-conventions';
+import {
+  ATTR_DB_SYSTEM,
+  DB_SYSTEM_VALUE_POSTGRESQL
+} from '@opentelemetry/semantic-conventions/incubating';
+
+console.log(ATTR_SERVICE_NAME); // 'service.name'
+console.log(ATTR_HTTP_ROUTE);   // 'http.route'
+
+// Bonus: the older exports did not include metric names from semconv.
+// 'http.client.request.duration' metric name
+console.log(METRIC_HTTP_CLIENT_REQUEST_DURATION);
+
+console.log(ATTR_DB_SYSTEM);    // 'db.system'
+// 'postgresql' enum value for 'db.system' attribute
+console.log(DB_SYSTEM_VALUE_POSTGRESQL);
+```
+
+**What is "incubating"?**
+The first three fields ('service.name', 'http.route', 'http.client.request.duration') are _stable_ in semantic conventions, the latter two are not. Besides changing the export names, `@opentelemetry/semantic-conventions@1.26.0` also separated stable and unstable into separate module entry points: stable conventions are imported `from '@opentelemetry/semantic-conventions'` and unstable `from '@opentelemetry/semantic-conventions/incubating'`. The name "incubating" was chosen to match the same name used by the Java and Python semantic-conventions packages.
+
+It is recommended that if you are using exports from _incubating_, that you **pin the version** in your package.json dependencies (e.g. via `npm install --save-exact @opentelemetry/semantic-conventions`), rather than using a caret dependency. This is because the removal of exports from the _incubating_ entry point is _not considered a breaking change_ and hence can happen in a minor version.
```

Comment:
```suggestion
It is recommended that if you are using exports from _incubating_, that you **pin the version** in your package.json dependencies (e.g. via `npm install --save-exact @opentelemetry/semantic-conventions`), rather than using a caret dependency. This is because the removal of exports from the _incubating_ entry point is _not considered a breaking change_ and hence can happen in a minor version. You may also consider duplicating these values into a hard-coded file of constants to allow updates to the semantic-conventions package in use.
```

Reply:
handled in commit 8fedc235e